<ui version="4.0" >
 <class>SvnPropDelDialog</class>
 <widget class="QDialog" name="SvnPropDelDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Delete Subversion Property</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="1" >
      <widget class="QLineEdit" name="propNameEdit" >
       <property name="toolTip" >
        <string>Enter the name of the property to be deleted</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2" >
      <widget class="QCheckBox" name="recurseCheckBox" >
       <property name="toolTip" >
        <string>Select to apply the property recursively</string>
       </property>
       <property name="text" >
        <string>Apply &amp;recursively</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="textLabel1" >
       <property name="text" >
        <string>Property &amp;Name:</string>
       </property>
       <property name="buddy" >
        <cstring>propNameEdit</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>propNameEdit</tabstop>
  <tabstop>recurseCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SvnPropDelDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>37</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel" >
     <x>37</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SvnPropDelDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>88</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel" >
     <x>88</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
